README.md

```diff
@@ -32,3 +28,152 @@
   1. Socket.io and Redis
   2. Popular modules
   3. Links
+
+## Introduction to NodeJS ##
+### CommonJS Modules ###
+Code in node apps is organized in modules. There are a couple of standards on how to write a module in JavaScript. By convention, NodeJS has decided to use the CommonJS Module Specification.
```

Comment:
NodeJS application organize their code into modules. There are a couple of standards on how to write a module in JavaScript. By convention, NodeJS has decided to use the CommonJS Module Specification.